doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -450,7 +450,11 @@ Multiprotocol Service Layer libraries
 Libraries for networking
 ------------------------
 
-|no_changes_yet_note|
+* :ref:`lib_nrf_cloud` library:
+
+  * Fixed:
+
+    * An issue that would cause resumption to fail when FOTA images are downloaded using the CoAP backend.
```

Comment:
Please remove this changelog entry and add a comment on the 2.9 release notes PR - https://github.com/nrfconnect/sdk-nrf/pull/19240.

Reply:
Took out the entry from the release notes.